build.gradle

```diff
@@ -143,6 +143,10 @@ allprojects {
         // see IntegrationRunner.stopOrphanedThreads
         jvmArgs "--add-opens=java.base/java.lang=ALL-UNNAMED"
 
+        // This is required for our dangling thread checks at the end of a connector run.
+        // see IntegrationRunner.stopOrphanedThreads
+        jvmArgs "--add-opens=java.base/java.lang=ALL-UNNAMED"
```

Comment:
this seems redundant considering the changes in the previous PR

Reply:
it's needed for tests. We have some tests that use the IntegrationRunner without docker. I moved it to the PR below though